### ISSUE TITLE
Support VS2022 Build Tools

### ISSUE THREAD

**Issue by jerone** [bazelbuild/bazel#16378]
### Description of the feature request:

Projects that are using Bazel, cannot upgrade to the latest version of [.NET 6](https://dotnet.microsoft.com/en-us/download/visual-studio-sdks).
For .NET 6 support for [Build Tools for Visual Studio **2022**](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022) is required.
Right now Bazel [support](https://bazel.build/install/windows#install-compilers) Build Tools for Visual Studio **2019**.

### What underlying problem are you trying to solve with this feature?

Projects that are using Bazel, cannot upgrade to the latest version of [.NET 6](https://dotnet.microsoft.com/en-us/download/visual-studio-sdks).

### Which operating system are you running Bazel on?

Windows

### Any other information, logs, or outputs that you want to share?

Pingback: https://github.com/SeleniumHQ/selenium/issues/10466

**Comment by meteorcloudy** [comment]
Bazel should work with VS2022, can you give a reproduce case for the failure case?

**Comment by sgowroji** [comment]
Hello @jerone, Could you please respond on the above [comment](https://github.com/bazelbuild/bazel/issues/16378#issuecomment-1282300013). Thanks!

**Comment by meteorcloudy** [comment]
@jerone Yes, the latest Bazel 5.3.2 should already support VS2022, we'll update the documentation soon, thanks for the report!
